airbyte-lib/airbyte_lib/registry.py

```diff
@@ -47,5 +48,7 @@ def get_connector_metadata(name: str) -> ConnectorMetadata:
     if not _cache:
         _update_cache()
     if not _cache or name not in _cache:
-        raise Exception(f"Connector {name} not found")
+        raise exc.AirbyteConnectorNotFoundError(
```

Comment:
This is using the same exception as the VenvExecutor if the binary to run the connector can't be found in the venv, but it's actually a very different kind of "not found" - this one might be a typo in the name, the other is a serious problem in the installation.

Reply:
Good catch! I've updated to the following:

```
        raise exc.AirbyteLibInputError(
            message="Connector name not found in registry.",
            guidance="Please double check the connector name.",
            context={
                "connector_name": name,
            },
        )

```